<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwindow</class>
 <widget class="QMainWindow" name="mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>910</width>
    <height>680</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Inventario</string>
  </property>
  <property name="windowOpacity">
   <double>0.990000000000000</double>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>631</width>
      <height>621</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>521</width>
       <height>591</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="lbl_titulo">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">
font: 24pt &quot;Sans Serif&quot;;</string>
          </property>
          <property name="text">
           <string>Inventario de biblioteca</string>
          </property>
          <property name="textFormat">
           <enum>Qt::RichText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_1">
          <property name="text">
           <string>Inventario</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_2">
          <property name="text">
           <string>Busqueda</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_3">
          <property name="text">
           <string>Modificar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_4">
          <property name="text">
           <string>Eliminar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <widget class="QPushButton" name="boton_agregarUsuario">
           <property name="geometry">
            <rect>
             <x>200</x>
             <y>170</y>
             <width>121</width>
             <height>27</height>
            </rect>
           </property>
           <property name="text">
            <string>Agregar Usuario</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="rs.qrc"/>
 </resources>
 <connections/>
</ui>
